tools/cobra/main.go

```diff
@@ -96,3 +108,13 @@ func ensureDirectory(dir string) error {
 
 	return nil
 }
+
+func header(filename string) string {
+	name := filepath.Base(filename)
```

Comment:
```suggestion
	name := filepath.Base(filename)
	name = strings.TrimSuffix(name, filepath.Ext(filename))
```
`.md` seems to still be in the title https://deploy-preview-660--cert-manager.netlify.app/docs/installation/components/acmesolver/

Reply:
Thanks Tim! Sorry I noticed your comment after updating the file